x-pack/plugins/synthetics/server/routes/monitor_cruds/add_monitor_project.ts

```diff
@@ -19,26 +19,25 @@ export const addSyntheticsProjectMonitorRoute: UMRestApiRouteFactory = (libs: UM
   validate: {
     body: schema.object({
```

Comment:
This I believe would cause validation issues when `keep_stale` continues to be passed. Discussed with @vigneshshanmugam that keep_stale would continue to be persisted, but if we want to go with removing logic from Kibana, we'll have to remove `keep_stale` from the synthetics agent as well, or get rid of this strict validation.